### DISCUSSION TITLE
仿生電子鼻

### DISCUSSION THREAD

**Discussion by eason8479** [1101-GS4719-C/Python-Community-GS4719-C#926] [General]
https://www.youtube.com/watch?v=TILjzuBGkRc

蠻驚訝的是居然只是外接模仿狗狗的特殊裝置，就能增加這麼多的效能

還有沒想到狗狗聞東西的時候還有這麼多細節

最後，沒想到世界上會有這樣的機構專門進行這方面的研究

**Comment by Evan0979** [comment]
可以錄影錄音了,不知道什麼時候可以錄味道,好期待哈哈哈

**Reply by eason8479** [reply]
不知道欸，不過有些電影/影片，可能還是會不要錄到氣味比較好
（ex:垃圾場、戰場、貧窮的社區）

**Comment by CPCpyt** [comment]
我覺得這個概念好酷，希望在日後能夠真正遇到這樣的技術！很想觀察看看！

**Comment by ruthchen1020** [comment]
很酷，如果氣味可以像影印機一樣被製作出來，那大家真的都不用去買香水了。
